src/test/groovy/org/prebid/server/functional/tests/CookieSyncSpec.groovy

```diff
@@ -1296,6 +1299,63 @@ class CookieSyncSpec extends BaseSpec {
         assert metric["cookie_sync.generic.tcf.blocked"] == 1
     }
 
+    def "PBS cookie sync with proper tcf parameter should process bidder sync and display metrics"() {
```

Comment:
Please, rename "tcf parameter" to smth more specific, for example "consent.tcfPolicyVersion"